docs/eigenda/operator-guides/upgrades/system-upgrades.md

```diff
@@ -6,20 +6,26 @@ sidebar_position: 2
 
-When performing a system upgrade, operators should be mindful of the following considerations: 
+When performing a system upgrade, operators should be mindful of the following considerations:
 - Maintain custody of BLS signing keys.
-- Ensure that the upgraded node remains reachable at the previously registered address. 
-- Maintain the integrity of all blob data stored by the node. 
+- Ensure that the upgraded node remains reachable at the previously registered address.
+- Maintain the integrity of all blob data stored by the node.
 
-## Maintaining Data Integrity
+## Node migration
 
-If you followed the setup steps in our guide to [running with docker](../run-a-node/run-with-docker/), then your node will store its data at the location specified by the `NODE_DB_PATH_HOST` in your `.env` file, which is bind-mounted to the EigenDA docker container. 
+If you followed the setup steps in our guide to [running with docker](../run-a-node/run-with-docker/), then your node will store its data at the location specified by the `NODE_DB_PATH_HOST` in your `.env` file, which is bind-mounted to the EigenDA docker container.
 
-If your system upgrade requires you to change this storage location, ensure that all of the original data stored by the DA node survives in the new location. 
+Generally speaking, suppose you want to migrate node to a new machine, you should follow the following sequence in order to maintain data integrity and remain meeting the [Protocol SLA](../requirements/protocol-SLA/) during and after the migration:
 
-Generally speaking, you should follow the following sequence in order to maintain data integrity: 
-1. Provision the new storage location. Suppose that the variable `$NODE_DB_PATH_HOST_NEW` points to your new storage location.
-2. Stop the DA node (e.g. `docker-compose down`)
-3. Copy the the contents of the old storage location to the new storage location (e.g. `cp -r $NODE_DB_PATH_HOST $NODE_DB_PATH_HOST_NEW`)
-4. Update the DA node configuration (e.g. set the `NODE_DB_PATH_HOST` variable in your `.env` file to the value of `$NODE_DB_PATH_HOST_NEW`).
-5. Restart the DA node (e.g. `docker-compose up -d`)
\ No newline at end of file
+**Old machine**:
+1. Backup the keys stored at the machine as well as other configs you want to migrate
+2. Opt-out of all the quorums
+3. Keep the node running for >1h (note after this the node will stop receiving dispersal requests)
+4. Continue to keep the node running (for retrieval traffic) while spinning up the new machine
+
+**New machine**:
+1. Copy over the files from old machine located at `NODE_DB_PATH_HOST`
+2. Start the EigenDA Node (e.g. `docker-compose up -d`) with the files copied from the old machine (the file should be placed under the path per `NODE_DB_PATH_HOST`) and make sure node is reachable
+3. Opt-in the quorums with a new IP address (so the old machine remains reachable with original IP while this is setting up)
```

Comment:
`with a new IP address`
If someone has DNS configured - they would just need to repoint DNS after the switch right? should we put that instructions too?

Reply:
Yes, just pointing the DNS to the new IP.